tests/Fixer/Operator/PostIncrementFixerTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Operator;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Kuba Werłos <werlos@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Operator\PostIncrementFixer
+ */
+final class PostIncrementFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        $preIncrementFixerTest = new PreIncrementFixerTest();
+
+        return array_map(
+            'array_reverse',
```

Comment:
that kind of invoking a function is undetectable by SCA tools, call it directly, not by name

Reply:
done, what about changing calls in whole code?

Reply:
:+1:, but as separated PR.
are there more calls like that? I tried to catch all of them :(

Reply:
Only 4 cases -> https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3080